test/test_functional_tensor_cleanup.py

```diff
@@ -0,0 +1,145 @@
+import unittest
+
+import torch
+import torchvision.transforms.functional_tensor as F_t
+
+
+class Tester(unittest.TestCase):
+    def setUp(self):
+        self.device = "cpu"
+
+    def test_get_image_size(self):
+        scripted_fn = torch.jit.script(F_t._get_image_size)
+
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor)
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+        shape = (100, 100)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        self.assertTrue([shape[-1], shape[-2]], scripted_fn(tensor))
+
+    def test_vflip(self):
```

Comment:
These tests are fairly repetitive. Consider rewriting them to eliminate the duplicate code.

For example you could define a `List[Tuple[callable, args]]` and loop through them.

Reply:
Updated as per your suggestion. please check and suggest improvements. :)

Reply:
Looks great.